cgmanifests/submodules/cgmanifest.json

```diff
@@ -335,7 +335,7 @@
       "component": {
         "type": "git",
         "git": {
-          "commitHash": "0c070abb0c40fec649f81a73a75b0098662ec486",
+          "commitHash": "a82c6a7010e2e332d8f74ad5b0c726fd47c85376",
           "repositoryUrl": "https://github.com/onnx/onnx"
```

Comment:
Should we have this diff and the one for TensorRT?

Reply:
Both onnx and tensorRT was updated in the past week without update this cgmanifest.json, this is actually add the missing piece

---
In reply to: [473612463](https://github.com/microsoft/onnxruntime/pull/4870#discussion_r473612463) [](ancestors = 473612463)

Reply:
Onnx was updated by #4791 and tensorRT was updated by #4780

Reply:
thanks for updating!